_extensions/quarto-gouv/_extension.yml

```
@@ -0,0 +1,10 @@
+title: Onyxia Presentation (light theme)
+author: Romain Lesur
+version: 1.1.0
+contributes:
+  formats:
+    docx:
+      reference-doc: ressources/template_lettre.docx
+      toc: false
+      number-sections: false
+      highlight-style: github
```

ajouter une ligne vide à la fin du fichier :

```suggestion
      highlight-style: github

```